Source/Scene/ShadowMap.js

```diff
@@ -939,7 +893,11 @@ define([
             if (enterFreezeFrame) {
                 // Recreate debug camera when entering freeze frame mode
                 shadowMap._debugCameraFrustum = shadowMap._debugCameraFrustum && shadowMap._debugCameraFrustum.destroy();
-                shadowMap._debugCameraFrustum = createDebugFrustum(shadowMap._sceneCamera, Color.CYAN);
+                shadowMap._debugCameraFrustum = new DebugCameraPrimitive({
+                    camera : shadowMap._sceneCamera,
+                    color : Color.CYAN,
+                    updateOnChange : false
```

Comment:
Do you have a use case for `updateOnChange : true`?

If not, should we leave it out for now?